### PR TITLE
[4.0] Namespace com_users

### PR DESCRIPTION
**Pull request by brianteeman**
Namespace of frontend components. Broken down into one PR for each component so that hopefully it can now be merged more easily and without the need for daily having to resolve merge conflicts.

### CODE REVIEW

File: components/com_users/Model/LoginModel.php [hunk 1]
```diff
@@ -30,7 +30,7 @@ class LoginModel extends FormModel
 	 * @param   array    $data      An optional array of data for the form to interogate.
 	 * @param   boolean  $loadData  True if the form is to load its own data (default case), false if not.
 	 *
-	 * @return  \JForm	A \JForm object on success, false on failure
+	 * @return  Form	A Form object on success, false on failure
```
Comment (wilsonge) [inline components/com_users/Model/LoginModel.php:34]: You need to add the `use` statement here for Form

File: components/com_users/Model/ProfileModel.php [hunk 2]
```diff
@@ -22,9 +23,6 @@
 use Joomla\CMS\Table\Table;
 use Joomla\Component\Users\Administrator\Model\UserModel;
 use Joomla\Registry\Registry;
-use Joomla\CMS\Factory;
-use Joomla\CMS\User\User;
-use Joomla\CMS\String\PunycodeHelper;
```
Comment (wilsonge) [inline components/com_users/Model/ProfileModel.php]: These 3 imports look like they are used

File: components/com_users/Model/RegistrationModel.php [hunk 3]
```diff
@@ -318,7 +318,7 @@ public function getData()
 	 * @param   array    $data      An optional array of data for the form to interogate.
 	 * @param   boolean  $loadData  True if the form is to load its own data (default case), false if not.
 	 *
-	 * @return  \JForm  A \JForm object on success, false on failure
+	 * @return  Form  A Form object on success, false on failure
```
Comment (wilsonge) [inline components/com_users/Model/RegistrationModel.php:322]: Missing `Form` import

**Review by wilsonge — DISMISSED** [review]
as per comments

File: components/com_users/Model/LoginModel.php (hunk 1 above)
Comment (brianteeman) [reply components/com_users/Model/LoginModel.php:34]: Done

File: components/com_users/Model/RegistrationModel.php (hunk 3 above)
Comment (brianteeman) [reply components/com_users/Model/RegistrationModel.php:322]: Done

**Comment by brianteeman** [comment]
Should all be done now - I did a bad job fixing merge conflicts

**Review by wilsonge — APPROVED** [review]

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**